.gitattributes

```diff
@@ -0,0 +1 @@
+packages/enty-prototype/* linguist-generated
```

Comment:
Is using linguist-generated for executable code a good idea?

Reply:
It's not published. It's just a scratch pad.

Reply:
I plan to remove it soon.